tests/manage/pv_services/test_dynamic_pvc_accessmodes_with_reclaim_policies.py

```diff
@@ -170,7 +160,20 @@ class TestRWODynamicPvc(BaseDynamicPvc):
     expected_pod_failure = 'Multi-Attach error for volume'
 
     @pytest.fixture()
-    def setup_base(self, request, interface_type, reclaim_policy):
+    def setup_base(
+        self,
+        request,
+        interface_type,
+        reclaim_policy,
+        rbd_secret_factory,
+        cephfs_secret_factory,
+        ceph_block_pool_factory,
+        project_factory
+    ):
+        self.rbd_secret_obj = rbd_secret_factory()
```

Comment:
@fbalak @clacroix12 
That's interesting usage :)
This will create everything in the setup or test itself?
(Sorry for repeating over and over on the same questions - this is new to me)

Reply:
This example will create the resources within the 'setup' since they are being executed in this fixture which the actual tests depend on.